src/server/common/helpers/errors.js

```diff
@@ -1,39 +1,34 @@
 import { statusCodes } from '~/src/server/common/constants/status-codes.js'
 
-/**
- * @param {number} statusCode
- */
 function statusCodeMessage(statusCode) {
-  switch (true) {
-    case statusCode === statusCodes.notFound:
+  switch (statusCode) {
+    case statusCodes.notFound:
       return 'Page not found'
-    case statusCode === statusCodes.forbidden:
+    case statusCodes.forbidden:
       return 'Forbidden'
-    case statusCode === statusCodes.unauthorized:
+    case statusCodes.unauthorized:
       return 'Unauthorized'
-    case statusCode === statusCodes.badRequest:
+    case statusCodes.badRequest:
       return 'Bad Request'
     default:
       return 'Something went wrong'
   }
 }
 
-/**
```

Comment:
I did see the other day, after an IJ update this stopped working.

This does work though and should give you the types:

```
/**
 * @param { import('@hapi/hapi').Request } request
 * @param { import('@hapi/hapi').ResponseToolkit } h
 */

```